nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-standard-record-utils/src/test/java/org/apache/nifi/schema/validation/TestStandardSchemaValidator.java

```diff
@@ -145,6 +148,9 @@ public void testValidateCorrectSimpleTypesStrictValidation() throws ParseExcepti
 
         valueMap.put("float_as_double", 8.0F);
 
+        valueMap.put("float_as_bigdecimal", 8.0F);
```

Comment:
Minor: we might want to add `"byte/short/int/long/bigint_as_bigdecimal"` as well.
Kind of redundant with the ones below but still better to have this stay a simplified universal test.

Reply:
The focus for this and the ones below is different: all of the ones added here is about the validation based on "narrow type", and the other cases cover the more specific situations, like allowing double as float in some cases.